aries_cloudagent/admin/server.py

```diff
@@ -249,6 +251,20 @@ def is_unprotected_path(path: str):
                 or path.startswith("/static/swagger/")
             )
 
+        @web.middleware
+        async def set_request_context(request, handler):
+            context = self.context.copy()
+
+            # This sets the context and message_router on the request
+            # This allows for different context per request, which
+            # is needed for multitenancy.
+            request["context"] = context
+            request["outbound_message_router"] = self.responder.send
+
+            return await handler(request)
+
+        middlewares.append(set_request_context)
```

Comment:
Even if multitenancy is not enabled we now set the conext on the request, this is to avoid endless if/else statements.